AK/URL.cpp

```diff
@@ -62,10 +62,10 @@ DeprecatedString URL::basename() const
     return percent_decode(last_segment);
 }
 
-// NOTE: This only exists for compatibility with the existing URL tests which check for both .is_null() and .is_empty().
+// NOTE: This only exists for compatibility with the existing URL tests which check for .is_empty().
 static DeprecatedString deprecated_string_percent_encode(DeprecatedString const& input, URL::PercentEncodeSet set = URL::PercentEncodeSet::Userinfo, URL::SpaceAsPlus space_as_plus = URL::SpaceAsPlus::No)
```

Comment:
We can actually remove this function entirely, see: https://github.com/SerenityOS/serenity/pull/21119/commits/1236dfe5e7168d6ad1c04cb96ff915040fe98cd1

Reply:
I yoinked that commit here, git rebase should handle it regardless of which PR gets merged first.